add obis-observer command GetShortName (#41)
jooby-dev/jooby-docs: docs/obis-observer/commands/GetShortName.md

# GetShortName

Request/response the short name list of the specific OBIS code.

## Request

### Format

| Size  | Type   | Field                     |
| ----- | ------ | ------------------------- |
| `1`   | `byte` | command id = `0x01`       |
| `1`   | `byte` | [request id](#request-id) |
| `3-7` | `byte` | [obis](#obis)             |

### Parameters

#### **request id**

> It's a unique number for pair device and GetShortName correction request.

Reviewer: ```suggestion
It's a unique number to link request with response.
```

> Then increment this value for each following request.

Reviewer: It should be incremented for each request.

Author: description for request id changed

It should be done to avoid double processing of the request.

#### **obis**

Obis code.
<br>
[See details](../types.md#obis).

### Examples

#### get short name for OBIS `0.9.1`

| Field      | Value | Hex          |
| ---------- | ----- | ------------ |
| command id | `1`   | `0x01`       |
| request id | `2`   | `0x03`       |
| obis       |       | `0x02000901` |

Message hex dump: `01 03 02 00 09 01`

## Response

### Format

| Size  | Type   | Field                        |
| ----- | ------ | ---------------------------- |
| `1`   | `byte` | command id = `0x02`          |
| `1`   | `byte` | command size (dynamic, `4+`) |
| `1`   | `byte` | [request id](#request-id)    |
| `3-7` | `byte` | [obis](#obis)                |
| `1`   | `byte` | short name `1`               |
| ...   | ...    | ...                          |
| `1`   | `byte` | short name `N`               |

### Parameters

#### **request id**

> It's a unique number for pair device and GetShortName correction request.

Reviewer: also would be nice to correct

Author: fixed
